src/config.js

```diff
@@ -70,6 +74,23 @@ const fromEnvOrConfig = (name, default_) => {
 };
 
 
+/**
+ * Deserialize a value that might be JSON, passing it thru if it isn't.
+ *
+ * @param {any} x - Value which might be JSON
+ */
+function maybeJSON(x) {
```

Comment:
_(minor)_

Is this for `OIDC_IAT_BACKDATED_BY`? If so I think it should be in 9644d8c9d946b554bdaa11370e491a4eb71030fb.

Reply:
Ah, no, it's for `OIDC_CONFIGURATION` which is introduced in the same commit. But yes, it's also used subsequently by `OIDC_CONFIGURATION`.

Reply:
Oh! I missed that as being optional since I was focused on changes to the config files and only saw `OIDC_IAT_BACKDATED_BY` as a non-string value. Makes sense now.